packages/@react-spectrum/searchwithin/test/SearchWithin.test.js

```diff
@@ -152,40 +152,87 @@ describe('SearchWithin', function () {
 
     expect(searchfield).not.toHaveAttribute('aria-required');
 
-    expect(searchfield).toHaveAttribute('aria-labelledby', `${label.id} ${group.id}`);
-    expect(group).toHaveAttribute('aria-labelledby', `${label.id} ${group.id}`);
+    expect(searchfield).toHaveAttribute('aria-labelledby', `${label.id} ${picker.id} ${picker.childNodes[0].id}`);
+    expect(group).toHaveAttribute('aria-labelledby', label.id);
 
     expect(searchfield.classList.contains('is-quiet')).toBeFalsy();
     expect(picker.classList.contains('spectrum-Dropdown--quiet')).toBeFalsy();
   });
+});
 
-  it('Should handle aria-labels without visible label', function () {
-    let {getByRole} = renderSearchWithin({label: undefined, 'aria-label': 'Aria Label'});
+describe('SearchWithin labeling', function () {
+  it('Searchwithin (no label); SearchField no aria-label; Picker no aria-label', function () {
```

Comment:
I don't think we need to specify anything beyond no label, lets add a comment to SearchWithin's slots context and just say we don't allow aria-label on SearchField or Picker
```suggestion
  it('no label - default', function () {
```

Reply:
Fixed these, thanks!